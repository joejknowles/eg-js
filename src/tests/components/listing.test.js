import Component from '../../components/listing';

import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

const shallowRenderer = ReactTestUtils.createRenderer();

it('renders listing as before', () => {
  const component = shallowRenderer.render(<Component />);
  expect(component).toMatchSnapshot();
});
