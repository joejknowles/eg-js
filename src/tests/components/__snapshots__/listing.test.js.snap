exports[`test renders listing as before 1`] = `
<div
  className="event-listing">
  <Unknown
    url={undefined} />
  <Link
    onlyActiveOnIndex={false}
    style={Object {}}
    to="event/undefined">
    <h3
      className="event-title" />
  </Link>
  <p>
    , at 
  </p>
</div>
`;
